k8s/zookeeper/Makefile

```diff
@@ -4,18 +4,16 @@ include ../var.Makefile
 include ../images.Makefile
 
 CHART_NAME := zookeeper
-APP_ID ?= $(CHART_NAME)
+APP_ID ?= $(CHART_NAME)-managed-updates
```

Comment:
We recently removed `-managed-updates` suffix from apps that don't support kalm - https://github.com/GoogleCloudPlatform/click-to-deploy/commit/9d3d57fd081f90bb7e981ca78c93f0468bfab30a#diff-b5f6a1881c8e5401fdc371b378316d23.